webapp/nextjs/app/editor/page.module.css

```
@@ -143,3 +143,313 @@
+  color: #999;
+}
+
+.commentBody {
+  color: #555;
+  line-height: 1.6;
+  font-size: 0.95rem;
+  margin-top: 0.5rem;
+  /* 箇条書きの行頭が枠に食い込まないように余白 + 折返し */
+  padding-left: 0.75rem;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)